### PR TITLE
import tool: hiding filter logic

### PR DESCRIPTION
**Pull request by manavortex**
# Import Tool: Clean up filtering logic

### CODE REVIEW

File: WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs [hunk 1]
```diff
@@ -178,6 +178,26 @@ private async Task<bool> ExportSingleAsync(ExportableItemViewModel item, FileSys
...
+    /// - .ent files (they should be exported via wscript instead)
+    /// - .app files (this isn't handled)
+    /// - .inkatlas files
+    /// </summary>
+    /// <param name="filePath">The file path as string</param>
+    /// <returns>true if this file should not be shown in the import browser</returns>
+    private static bool ShouldFileBeFiltered(string filePath) =>
+        Path.GetExtension(filePath) switch
+        {
+            ".inkatlas" => // we're not handling this at the moment
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs:193]: Pretty sure it is handled

File: WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs [hunk 2]
```diff
@@ -178,6 +178,26 @@ private async Task<bool> ExportSingleAsync(ExportableItemViewModel item, FileSys
...
+                true,
+            ".ent" => // .ent files - export them via script
+                true,
+            ".app" => // .app files
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs:197]: Yeah, already handled with `CanExport`

File: WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs [hunk 3]
```diff
@@ -178,6 +178,26 @@ private async Task<bool> ExportSingleAsync(ExportableItemViewModel item, FileSys
...
+    /// - .inkatlas files
+    /// </summary>
+    /// <param name="filePath">The file path as string</param>
+    /// <returns>true if this file should not be shown in the import browser</returns>
+    private static bool ShouldFileBeFiltered(string filePath) =>
+        Path.GetExtension(filePath) switch
+        {
+            ".inkatlas" => // we're not handling this at the moment
+                true,
+            ".ent" => // .ent files - export them via script
```
Comment (seberoth) [inline WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs:195]: Also do this via `CanExport` then? Or is the import of it needed?

Comment (manavortex) [reply WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs:195]: OK, I'm gonk :D I'll look up CanExport!

**Pull request merged**